app/reducers/index.js

```diff
@@ -0,0 +1 @@
+export {default} from 'ember-redux/reducers/index';
```

Comment:
Previously, did someone using `ember-redux` always have to create a `reducers/index.js` file in order to prevent their app from breaking?